CUJExecutor/back

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# INPUT: a .prof file followed by any number of tags in {HSPL, HPL, PL}
```

Comment:
Technically, the tags are:

  - H
  - S
  - P

A method in the profile can be prefixed with any or all of them, in any order.  But it's okay (though not safe) to assume that the tags only come together as `HSPL`, `HPL`, or `PL`.

`L` is not a flag and it is the start of a Java class signature that ends with `;`.

Reply:
It makes sense for ART to only save certain tags to the profile, hence the lack of other combinations.